version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.28.1"
+version = "0.13.0-SNAPSHOT"
```

Comment:
It should work like this currently, but it's a bit safer to keep that `in ThisBuild`.

Reply:
That was an incomplete revert, sorry! With https://github.com/totti4ever/docspell/commit/8d3549e4022f6f7dbc829bf5129de77623c0a56a it should be correct again now (never intended to change anything) and I cannot see it under changes in this PR anymore

Reply:
ah ok